docs/pages-for-subheaders/about-provisioning-drivers.md

```diff
@@ -23,12 +23,6 @@ By default, Rancher has activated several hosted Kubernetes cloud providers incl
 *  [Google GKE](../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/gke.md)
 *  [Azure AKS](../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md)
 
-There are several other hosted Kubernetes cloud providers that are disabled by default, but are packaged in Rancher:
-
-* [Alibaba ACK](../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/alibaba.md)
-* [Huawei CCE](../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/huawei.md)
-* [Tencent](../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/tencent.md)
```

Comment:
We still have these listed on the KontainerDrivers page of the Rancher UI. I think we need to add some language advising against using them unless and until they're removed. Otherwise, we might get requests to restore the how-to guides we're removing in this PR.